modules/admin_manual/pages/appliance/troubleshooting.adoc

```diff
@@ -0,0 +1,11 @@
+= Troubleshooting
+
+If you have encountered an issue, here is what support needs in order to get a quick resolution of your issue:
```

Comment:
I suggest the sentence end with "_in order to provide a rapid resolution to your issue:_"

Reply:
Sounds good